modules/terms/partials/admin-api.adoc

```diff
@@ -1,8 +1,10 @@
 === Admin API
 :term-name: Admin API
-:hover-text: A REST API used to manage and monitor Redpanda clusters. It uses the default port 9644.
+:hover-text: A REST API used to manage and monitor Redpanda Self-Managed clusters. It uses the default port 9644.
 :category: Redpanda features
 
+ifndef::env-cloud[]
```

Comment:
Do we need conditionalizing for this glossterm? I wouldn't expect to use it in Cloud

Reply:
Admin API not supported in Cloud